charts/ingress-nginx/values.yaml

```diff
@@ -768,22 +768,24 @@ revisionHistoryLimit: 10
 ##
 defaultBackend:
   ##
-  enabled: false
+  enabled: true
 
   name: defaultbackend
   image:
-    registry: registry.k8s.io
-    image: defaultbackend-amd64
+    repository: nginx
+    tag: alpine
     ## for backwards compatibility consider setting the full image url via the repository value below
     ## use *either* current default registry/image or repository format or installing chart by providing the values.yaml will fail
     ## repository:
-    tag: "1.5"
     pullPolicy: IfNotPresent
     # nobody user -> uid 65534
-    runAsUser: 65534
-    runAsNonRoot: true
-    readOnlyRootFilesystem: true
-    allowPrivilegeEscalation: false
+    # runAsUser: 65534
+    # runAsNonRoot: true
+    # readOnlyRootFilesystem: true
+    # allowPrivilegeEscalation: false
```

Comment:
This is a potential security issue and unnecessary I believe.

Reply:
nginx:1.23.1-alpine For the moment, in the future we will create the backend with Go